OTAnalytics/plugin_ui/customtkinter_gui/toplevel_flows.py

```diff
@@ -91,6 +107,7 @@ def _set_close_on_return_key(self) -> None:
     def close(self, event: Any = None) -> None:
         if not self._sections_are_valid():
             return
+        self.input_values[FLOW_ID] = self._current_id.get()
```

Comment:
You dont necessarily need the `_current_id` property. You can just do `self.entry_id.get()`.